src/controllers/product.js

```diff
@@ -235,11 +235,53 @@ const orderRequest = async (req, res) => {
   }
 };
 
+const getRequesters = async (req, res) => {
+  try {
+    const product = await ProductModel.findById(req.params.productId).populate(
+      'orderRequests',
+      'firstName lastName username email phoneNumber'
+    );
+    if (!product) {
+      return res.status(404).json({
+        message: 'Product not found',
+      });
+    }
+
+    if (String(product.publisher) !== req.user.userId) {
+      return res.status(401).json({
+        message: 'You are not authorized to perform this action',
+      });
+    }
+
+    if (product.postType !== 'Donate') {
+      return res.status(400).json({
+        message: 'This is not a donation product: no requesters',
+      });
+    }
+
+    if (product.orderRequests.length === 0) {
+      return res.status(400).json({
+        message: 'No requesters found for this product',
+      });
+    }
+
+    const requesters = {
+      ID: product._id,
```

Comment:
why the keys here start with capital letter?

Reply:
It's the info shown to the user:
![Screenshot from 2022-08-24 11-40-50](https://user-images.githubusercontent.com/102853969/186374316-274504c5-50f9-48ed-92d1-3fc120cbb3d3.png)

Should I change it @halitbatur?